.gitignore

```diff
@@ -3,6 +3,7 @@ dist/
 docs/**/*.html
 docs/.nojekyll
 docs/assets
+test*
```

Comment:
Is this needed?

Reply:
I used few test files always prefixed by "test" to check my change, one time unit-test added we can remove it!